index.js

```diff
@@ -4,7 +4,7 @@ var loaderUtils = require('loader-utils');
 module.exports = function(source) {
   this.cacheable && this.cacheable();
   var query = loaderUtils.parseQuery(this.query);
-  var options = this.options.ejsLoader || {};
+  var options = this.options ? this.options.ejsLoader || {} : {};
```

Comment:
I'd prefer this to be a breaking change instead of trying to be backward compatible.

Reply:
Can you explain why we should break and support one version for webpack 3 and one for webpack 4 if its not neccessary?